Base/res/html/misc/border-radius.html

```diff
@@ -274,6 +274,11 @@ <h3>Border-radius</h3>
     <br>
     <br/>
 
+    <p>This is an <code>&lt;img&gt;</code> tag with a border radius</p>
```

Comment:
since you say `an <img>` here, you should also do that in the commit message :)

Reply:
Change the commit message to be consistent with the previous examples commit, so still no `an` :)